controller/Makefile

```diff
@@ -4,6 +4,8 @@ include ../includes.mk
 
 all: build run
 
+SHELL_SCRIPTS = build.sh $(wildcard bin/*) $(wildcard migrations/data/*.sh)
```

Comment:
good point. It'll remove the wildcard for the etcd data migrations, at the very least :)